microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -223,4 +223,13 @@ For the latest images included with {microshift-short}, view the contents of the
 [id="microshift-4-16-3-bug-fixes_{context}"]
 ==== Bug fixes
 
-* A mistake in validation allowed empty entries in the `listenAddress` configuration parameter. The empty entries prevented {microshift-short} from starting and rendered a difficult-to-understand error message. Empty entries in the list are now filtered.
\ No newline at end of file
+* A mistake in validation allowed empty entries in the `listenAddress` configuration parameter. The empty entries prevented {microshift-short} from starting and rendered a difficult-to-understand error message. Empty entries in the list are now filtered.
```

Comment:
```suggestion
* An error in validation allowed empty entries in the `listenAddress` configuration parameter. The empty entries prevented {microshift-short} from starting and rendered a difficult-to-understand error message. Empty entries in the list are now filtered.
```

Reply:
Huh, that's last week's release note. Interesting how it popped up in the diff. Fix it anyway! TY!